<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAnnotationWidget</class>
 <widget class="QWidget" name="CreateAnnotationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>218</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>58</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create annotations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Save annotation(s) to</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="existingObjectRB">
        <property name="text">
         <string>Existing annotation table</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="existingObjectCombo"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="existingObjectButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../U2Gui.qrc">
          <normaloff>:/core/images/load_selected_documents.png</normaloff>:/core/images/load_selected_documents.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="newFileRB">
        <property name="text">
         <string>Create new table</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="newFileEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="newFileButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Annotation parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="groupNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Group name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="groupNameEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="groupNameButton">
        <property name="toolTip">
         <string>Predefined group names</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../U2Gui.qrc">
          <normaloff>:/core/images/group_auto.png</normaloff>:/core/images/group_auto.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="annotationNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Annotation name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="annotationNameEdit">
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="showNameGroupsButton">
        <property name="toolTip">
         <string>Predefined annotation names</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../U2Gui.qrc">
          <normaloff>:/core/images/predefined_annotation_groups.png</normaloff>:/core/images/predefined_annotation_groups.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="locationEdit">
        <property name="toolTip">
         <string>Annotation location in GenBank format</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="complementButton">
        <property name="toolTip">
         <string>Add/remove complement flag</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../U2Gui.qrc">
          <normaloff>:/core/images/do_complement.png</normaloff>:/core/images/do_complement.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>groupNameEdit</tabstop>
  <tabstop>annotationNameEdit</tabstop>
  <tabstop>locationEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../U2Gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>newFileRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>newFileEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newFileRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>newFileButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>existingObjectRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>existingObjectCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>existingObjectRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>existingObjectButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
